GPU/GLES/Framebuffer.cpp

```diff
@@ -973,6 +974,19 @@ struct CardboardSettings * FramebufferManager::GetCardboardSettings(struct Cardb
 void FramebufferManager::CopyDisplayToOutput() {
 	fbo_unbind();
 	glstate.viewport.set(0, 0, pixelWidth_, pixelHeight_);
+
+	if (useBufferedRendering_) {
+		glClearColor(0.0f, 0.0f, 0.0f, 0.0f);
```

Comment:
Do we need to flush (used by glClear):
 * scissor test
 * colormask

Before issuing this clear?  Why not use `ClearBuffer()`?

-[Unknown]